src/init/features/apphosting/secrets.ts

```diff
@@ -73,6 +76,7 @@ export async function grantSecretAccess(
   }
 
   try {
+    // TODO: Merge with existing bindings with the same role
```

Comment:
This is intentional, IAM will dedupe/coalesce roles fo rus

Reply:
Would like to verify. If so, I believe this is new.

Reply:
I can confirm I verified this manually when testing the CLI

You can see that in the original version of the commit there was an `updateBindings` method https://github.com/firebase/firebase-tools/pull/6899/commits/0fb470c874e9c7a31c56d623af8195080f098909#diff-1de7c7b0fb2e58db23edd1a7905f884d38b83d93d51ed5edd39a0c082cc3de49L78-L98